README.md

```diff
@@ -1,171 +1,48 @@
-  ```
-
-- Set appropriate permissions.
-
-  ```
-  sudo chown -R $USER:$GROUP ~/.npm
-  sudo chown -R $USER:$GROUP ~/.config
-  ```
-
-## Installation Steps
+## Installation
 
 Clone the Lisk Explorer Repository:
 
 ```
 git clone https://github.com/LiskHQ/lisk-explorer.git
 cd lisk-explorer
-npm install
-```
-
-## Build Steps
-
-#### Frontend
- The frontend is using Webpack to create core bundles for Lisk Explorer.
-
- For having a watcher to generate bundles continuously for all the changes of the code, Run the following command:
-
-`npm run watch`
-
- And for generating the minified bundles in production environment run:
-
-`npm run build`
-
- If you want to add a meta tag with name and content defined (For example to verify your ownership to Google analytics) run:
-
- `SERVICE_NAME='your service name' CLIENT_ID='you client id' npm run build`
-
-## Post-deployment Actions
-
-#### Market Watcher
- Candlestick data needs to be initialized prior to starting Lisk Explorer. During runtime candlestick data is updated automatically.
-
-This step writes data to the local Redis instance. Make sure that your application is already deployed and has access to the production Redis database.
-
-To build candlestick data for each exchange run:
-
-`grunt candles:build`
-
-To update candlestick data manually run after initialization:
-
-`grunt candles:update`
-
-## Configuration
-
-The default `config.js` file contains all of the configuration settings for Lisk Explorer. These options can be modified according to comments included in configuration file.
-
-## Managing Lisk Explorer
-
-To test that Lisk Explorer is configured correctly, use the following commands:
-
 ```
-npm run start          // default config based on environment variables
-npm run start:local    // connect with localhost:4000
-npm run start:testnet  // connect with testnet
-npm run start:mainnet  // connect with mainnet
-```
-
-Open: <http://localhost:6040>, or if its running on a remote system, switch `localhost` for the external IP Address of the machine.
-
-Once the process is verified as running correctly, `CTRL+C` and start the process with `PM2`. This will fork the process into the background and automatically recover the process if it fails.
 
-`pm2 start pm2-explorer.json`
+### Installation with Docker (recommended)
 
-After the process is started its runtime status and log location can be found by issuing this statement:
+In order to install Docker, refer to the offcial [Docker Installation Instruction](https://docs.docker.com/install/).
 
-`pm2 list`
-
-To stop Explorer after it has been started with `PM2`, issue the following command:
-
-`pm2 stop lisk-explorer`
-
-## Docker
-
-### Using docker-compose
+You will also need Docker Compose [Install Docker Compose](https://docs.docker.com/compose/install/).
 
 Update `docker-lisk-core.env` to choose your preferred node. You can easily switch between Mainnet and Testnet nodes by changing content of the env file.
```

Comment:
needs to be updated analog to docker setup page

Reply:
The new readme does not mention this method